indra/llui/llfloater.cpp

```diff
@@ -3419,12 +3419,12 @@ bool LLFloater::isVisible(const LLFloater* floater)
     return floater && floater->getVisible();
 }
 
-bool LLFloater::buildFromFile(const std::string& filename)
+bool LLFloater::buildFromFile(const std::string& filename, bool cacheable)
```

Comment:
Might be good idea to add a check from single instance floaters and either warn or override value with false when both 'cacheable' and single instance. Because both at the same time make no sense.

Reply:
Done